packages/framework/src/servers/nest/nest.utils.ts

```diff
@@ -0,0 +1,9 @@
+import { Type } from '@nestjs/common';
+
+export function applyDecorators<T>(baseClass: Type<T>, decorators: Array<ClassDecorator> = []): Type<T> {
+  return decorators.reduce((decoratedController, decorator) => {
+    const result = decorator(decoratedController);
```

Comment:
I think it can be more generic than referencing `controller`

```suggestion
  return decorators.reduce((decoratedClass, decorator) => {
    const result = decorator(decoratedClass);
```